1_The_Physical/Level_1_Stateless_Guess_Point/1_Arrange_Act_Assert_Backwards/2_1_Password_Validator/src/index.ts

```diff
@@ -0,0 +1,37 @@
+export default class PasswordValidator {
+    static validate(password: string): { result: boolean, errors: string[] } {
```

Comment:
I'd prefer to extract the type like, and use it instead of using the type directly
```ts
type Result = {
  result: boolean;
  errors: string[];
}